app/helpers.php

```diff
@@ -33,6 +34,18 @@ function edit_image($image, $coords)
 }
 
 /**
+ * Query for total reactions for a post.
+ *
+ * @param int $postableId
+ * @param int $postableType
+ * @return int total count
+ */
+function getTotalPostReactions($postableId, $postableType)
+{
+    return Reaction::where(['postable_id' => $postableId, 'postable_type' => $postableType])->count();
```

Comment:
This will make a separate "count" query for each model in the results (so if there are 5 posts returned from a request, it'll make 5 separate SQL queries to get each of their reactions). This is pretty rough from a performance standpoint (imagine if there were 50 posts being requested!)

Try looking into querying the posts [with the count of reactions](https://laravel.com/docs/5.4/eloquent-relationships#counting-related-models).